docs/modules/ROOT/pages/limitations.adoc

```diff
@@ -5,3 +5,42 @@ The Stackable Agent has limitations, some of which are documented here.
 * Kubernetes v1.19+ is required.
 * The maximum number of pods supported by the agent is currently hardcoded to 110.
   This restriction can be lifted if the need arises.
+* SSL certificate is not automatically renewed due to upstream bug https://github.com/krustlet/krustlet/issues/553 (manual activity required described below)
+
+== SSL certificate renewal ==
+
+The agent SSL certificate expiry date can be observed via `openssl x509 -enddate -noout -in /etc/stackable/stackable-agent/secret/agent.crt`.
+
+See upstream documentation about the certificate system at https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/, and the following steps:
+
+To generate a new certificate, first delete any current CSR:
+
+```
+kubectl delete csr/${HOSTNAME}-tls
+```
+
+If you have a CSR already, the agent will fail and log in to the journal (`journalctl -u stackable-agent`):
+
+```
+Sep 09 13:51:59 server1 stackable-agent[15457]: Caused by:
+Sep 09 13:51:59 server1 stackable-agent[15457]:     certificatesigningrequests.certificates.k8s.io "server1-tls" already exists: AlreadyExists
+```
+
+Remove the current certificate file and restart the agent:
+
+```
+sudo rm /etc/stackable/stackable-agent/secret/agent.crt && sudo systemctl restart stackable-agent
```

Comment:
nit: I'd prefer to spread this over two lines, just to be very clear that this is two distinct actions.

Reply:
I think we want to only do the restart if we did successfully remove the file, so I'll put over two lines but keep the &&. If you still prefer two distinct lines, I'll do that instead.

Reply:
That is a fair point, didn't think of that.